configure.in

```diff
@@ -5947,10 +5950,8 @@ if test "$wxUSE_SOCKETS" = "yes"; then
 fi
 
 if test "$wxUSE_SOCKETS" = "yes" ; then
-    dnl this test may be appropriate if building under cygwin
-    dnl right now I'm assuming it also uses the winsock stuff
-    dnl like mingw does..  -- RL
-    if test "$TOOLKIT" != "MSW"; then
+    dnl under Windows we always have sockets
```

Comment:
This comment doesn't seem to be appropriate here, it should probably say "We don't use getsockname()" under Windows or something like that.